### ISSUE TITLE
Package won't work with symfony/workflow < 4.1

### ISSUE THREAD

**Issue by kblais** [brexis/laravel-workflow#57]
While trying to figure an issue I have with the package, I realized it can't work with symfony/workflow < 4.1 since 1.3.0 because of the use of `InstanceOfSupportStrategy` here : https://github.com/brexis/laravel-workflow/blob/f09800a55788f834a347e4c904a50d1c786a32f9/src/WorkflowRegistry.php#L78

This class was introduced in symfony/workflow 4.1 to replace the now-deprecated `ClassInstanceSupportStrategy`.

So either we define the minimum requirement to `"symfony/worflow": "^4.1"`, or we add a compatibility layer to check if `InstanceOfSupportStrategy` exists, and if not use the `ClassInstanceSupportStrategy`.

What do you think about this ?

**Comment by brexis** [comment]
Yes, Use the previous versions before 1.3.0 for with Symfony Workflow < 4.1.

**Comment by kblais** [comment]
Well, then the composer.json should be updated to only use Symfony components >= 4.1, no ? Because right now, you can require the package, but it won't work (which is pretty frustrating when you think about it)
